Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM nartayzhanybekov/tchaikovski:latest
+# Create app directory
+WORKDIR /app
+COPY ./http_server.py /app
+# Install app dependencies
```

Comment:
Do we really need to do all of this stuff? Isn't it already declared in backend/Dockerfile and we can just specify the FROM and CMD?

Reply:
https://hub.docker.com/layers/nartayzhanybekov/tchaikovski/latest/images/sha256-32b3e13caba7c952dbaf1fbe8a434c4a67850ebee989c6fc4f379d9d62ad1a0a?context=explore